optuna/samplers/_gp/sampler.py

```diff
@@ -388,6 +417,9 @@ def sample_independent(
         param_name: str,
         param_distribution: BaseDistribution,
     ) -> Any:
+        if self._warn_independent_sampling:
+            self._log_independent_sampling(trial, param_name)
```

Comment:
A warning should be displayed when sampling independently and the number of trials are sufficient for the same reason  as previous comment
```suggestion
        if self._warn_independent_sampling:
            states = (TrialState.COMPLETE,)
            trials = study._get_trials(deepcopy=False, states=states, use_cache=True)
            if len(trials) >= self._n_startup_trials:
                self._log_independent_sampling(trial, param_name)
```